src/modules/LanItems.rb

```diff
@@ -2336,6 +2336,8 @@ def createS390Device(rule)
     #  @param [Array] list of hashes describing interfaces in AY profile
     #  @return [Array] list of hashes for udev rules
     def createUdevFromIfaceName(interfaces)
+      return [] if !interfaces || interfaces.empty?
```

Comment:
It's the fastest exit from the method, which easy to understand and independent on the rest of the code even in the future